src/stdatamodels/jwst/datamodels/schemas/miri_lrs_psf.schema.yaml

```diff
@@ -0,0 +1,44 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/miri_lrs_psf.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_readpatt.schema
+- $ref: keyword_psubarray.schema
+- $ref: keyword_grating.schema
```

Comment:
Should this be keyword_filter?
The example file on CRDS has:
`META.INSTRUMENT.FILTER	P750L`
but I don't see a defined `META.INSTRUMENT.GRATING`.

Reply:
Yes it probably should include filter.